doc/sphinx/Pacemaker_Remote/kvm-tutorial.rst

```diff
@@ -75,7 +75,7 @@ replacing the cluster name and hostname as desired:
 
 .. code-block:: none
 
-    # pcs cluster setup --force --local --name mycluster example-host
+    # pcs cluster setup mycluster example-host
```

Comment:
Similar to my comments on the remote walk-through, I think we can do a similar "Configure Cluster Nodes" here and skip everything before Install Virtualization Software.

For example purposes, we should use the same cluster for both chapters (i.e. either node1/node2 or pcmk-1/pcmk-2 ... probably using the same names as Clusters from Scratch makes sense, in case someone wants to do that first then this one).

In CentOS Stream 8, there is a "pcs cluster node add-guest" command similar to add-remote in that it creates the VirtualDomain resource, creates and copies the key, and enables pacemaker_remote, so those steps do not need to be done here (but are worth demonstrating with pcs cluster status, ls -l /etc/pacemaker and systemctl status pacemaker_remote similar to the remote walk-through).